src/components/NavBar.jsx

```diff
@@ -1,66 +1,104 @@
-    const [button, setButton] = useState(true);
+  const [click, setClick] = useState(false);
+  const [button, setButton] = useState(true);
 
-    const handleClick = () => setClick(!click);
-    const closeMobileMenu = () => setClick(false);
+  const handleClick = () => setClick(!click);
+  const closeMobileMenu = () => setClick(false);
 
-    const showButton = () => {
-        if(window.innerWidth <= 960) {
-            setButton(false);
-        } else {
-            setButton(true);
-        }
-    };
+  const showButton = () => {
+    if (window.innerWidth <= 960) {
+      setButton(false);
+    } else {
+      setButton(true);
+    }
+  };
 
-    useEffect(() => {
-        showButton();
-    }, []);
+  function scrollToSection(sectionId) {
+    const scrollTarget = document.getElementById(sectionId);
 
-    window.addEventListener('resize', showButton);
+    //window.scrollTo(0, findPosition scrollTarget));
 
-    return (
-        <>
-        <nav className='navbar'>
-            <div className="navbar-container">
-                <Link to="/" className="navbar-logo" onClick={closeMobileMenu}>
-                    Capathon <i className="fa-solid fa-code"></i>
-                </Link>
-                <div className="menu-icon" onClick={handleClick}>
-                    <i className={click ? 'fas fa-times' : 'fas fa-bars'} />
-                </div>
-                <ul className={ click ? 'nav-menu active' : 'nav-menu'}>
-                    <li className="nav-item">
-                        <Link to='/' className='nav-links' onClick={closeMobileMenu}> 
-                            Page id
-                        </Link>
-                    </li>
-                    <li className="nav-item">
-                        <Link to='/' className='nav-links' onClick={closeMobileMenu}>
-                            Page id 1
-                        </Link>
-                    </li>
-                    <li className="nav-item">
-                        <Link to='/' className='nav-links' onClick={closeMobileMenu}>
-                            Page id 2
-                        </Link>
-                    </li>
-                    <li>
-                        <Link to='/'className='nav-links-mobile'onClick={closeMobileMenu}>
-                            Other Type
-                        </Link>
-                    </li>
-                </ul>
-                {button && <Button buttonStyle="btn--outline">Sign Up</Button>}
-            </div>
-        </nav>
-        </>
-    );
+    console.log(scrollTarget);
+
+    scrollTarget?.scrollIntoView({
+      behavior: "smooth",
+    });
+  }
+
+  function findPosition(obj) {
+    var currentTop = 0;
+    if (obj.offsetParent) {
+      do {
+        currentTop += obj.offsetTop;
+      } while ((obj = obj.offsetParent));
+      return [currentTop];
+    }
+  }
+
+  useEffect(() => {
+    showButton();
+  }, []);
+
+  window.addEventListener("resize", showButton);
+
+  return (
+    <>
```

Comment:
This is called a react fragment `<> code ... </>` which you're correctly using elsewhere, but not here. Because the point of such a fragment is to group elements that are siblings to eachother but they don't share a parent element that serves as a root. But this component **does** have a root element, which is the `<nav>` element so you can remove it here.